gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/reactive/core/v4/analytics/LoggingContext.java

```diff
@@ -115,22 +123,24 @@ public void setMaxSizeLogMessage(String maxSizeLogMessage) {
         }
     }
 
-    public String getExcludedResponseTypes() {
-        return excludedResponseTypes;
-    }
-
-    public void setExcludedResponseTypes(final String excludedResponseTypes) {
-        this.excludedResponseTypes = excludedResponseTypes;
-    }
-
     public boolean isContentTypeLoggable(final String contentType) {
         // init pattern
         if (excludedContentTypesPattern == null) {
-            try {
-                excludedContentTypesPattern = Pattern.compile(excludedResponseTypes);
-            } catch (Exception e) {
-                excludedContentTypesPattern = Pattern.compile(DEFAULT_EXCLUDED_CONTENT_TYPES);
-            }
+            this.excludedContentTypesPattern = Stream.of(
+                logging.getOverrideContentTypeValidation(),
+                excludedResponseTypes,
+                DEFAULT_EXCLUDED_CONTENT_TYPES
+            )
+                .filter(pattern -> pattern != null && !pattern.isEmpty())
+                .flatMap(pattern -> {
+                    try {
+                        return Stream.of(Pattern.compile(pattern));
+                    } catch (PatternSyntaxException e) {
+                        return Stream.empty();
```

Comment:
We should warn someone that the pattern provided can't be compiled. I fear that we could have someone that messes with the regex, and the result will be that no content will be logged (which is safe,) but how can we inform users that they have to fix the value 🤔